mllib/src/main/scala/org/apache/spark/mllib/evaluation/RegressionMetrics.scala

```diff
@@ -109,6 +111,14 @@ class RegressionMetrics @Since("2.0.0") (
   }
 
   /**
+   * @return The root mean squared log error.
+   */
+  @Since("2.0.0")
+  def rootMeanSquaredLogError: Double = {
```

Comment:
@srowen Thanks for the comments. I have made the first two changes you asked for. On the extra computation side, would you rather prefer to have second pass over the data only if the RootMeanSquaredLogError has been called? If so, I would be happy to push that change. Or if you want to suggest any better solution, it would be interesting to know.